kebechet/config.py

```diff
@@ -35,6 +35,12 @@
 class _Config:
     """Library-wide configuration."""
 
+    # This is used to generate service urls from slugs.
+    _SERVICE_URLS = {
+        "github": {"service_url": "https://github.com/{slug}"},
+        "gitlab": {"service_url": "https://github.com/{slug}"},
```

Comment:
Is there needed the nested dict?

```python
    _SERVICE_URLS = {
        "github": "https://github.com/{slug}",
        "gitlab":  "https://github.com/{slug}",
    }
```

Reply:
no, I dont think so, it could be an list? ... and there is a typo in the gitlab url

Reply:
These are defaults if the host of service is not explicitly provided (for non-internal/custom deployments).

Reply:
I changed the typo. This was followed in the services.py so I thought I would use the same pattern, should I change it to something else?